### PR TITLE
read map from file

### PR DESCRIPTION
**Pull request by sousajo-cc**

### CODE REVIEW

File: src/state/game/map.rs [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 use super::positions::Positions;
 
 use crate::sprite::{SpriteType, SpriteTypeStates, SpriteVariant};
 use crate::Timer;
+use crate::log::console_log;
+
+const LEVEL_00: &str = include_str!("../../../assets/maps/00_map.txt");
+const LEVEL_01: &str = include_str!("../../../assets/maps/01_map.txt");
+const LEVEL_02: &str = include_str!("../../../assets/maps/02_map.txt");
+const LEVEL_03: &str = include_str!("../../../assets/maps/03_map.txt");
```
Comment (luizchagasjardim) [inline src/state/game/map.rs:14]: I know your pain. Trying to somehow point to the assets folder from a variable.

File: assets/maps/00_map.txt [hunk 2]
```diff
@@ -0,0 +1,100 @@
+Tile::Ground Tile::Ground Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty Tile::Empty
```
Comment (luizchagasjardim) [inline assets/maps/00_map.txt]: The prefix "Tile::" could be omitted, since it's in every tile.

**Review by luizchagasjardim — APPROVED** [review]
Very good. I'm anxious to see the positions also being read from a file.

**Comment by luizchagasjardim** [comment]
@sousajo-cc is this ready? can you merge it?

**Comment by sousajo-cc** [comment]
sure thing done

**Pull request merged**